### ISSUE TITLE
User Story: accept invite to org/team and be prompted to sign up for AllSpice

### ISSUE THREAD

**Issue by allthedoll** [allthedoll/allspice-product#59]
## TL;DR

- As a user that isn't signed up for AllSpice I want to accept an invitation to an org or team and create and account so that I can accept the invitation and use AllSpice.

## Product requirements TL;DR

- [ ] User added via email receives email to join org/team
  - [ ] User can click accept link to join org/team and it asks them to create an AllSpice account
  - [ ] They validate their account is not a robot
  - [ ] They are prompted to accept the invitation to the org/team
  - [ ] Website shows they are now a part of the org/team  
  - [ ] They receive a `Getting Started with AllSpice` email to help them get started

<img width="780" alt="Screenshot 2023-10-06 at 12 53 23 PM" src="https://github.com/allthedoll/allspice-product/assets/5758031/b291fce2-16fd-4cc9-88d9-73bfa61fcd1d">

## Design requirements

- TBD

## Technical requirements

- TBD

## Timeframe

-  Noted in [onboarding initiative](https://docs.google.com/document/d/1N5YY-I72F4WMXZt2NSJVvB2eSypoLuMATuKbO_rVuQs/edit?usp=sharing)
